src/background/customDomain.ts

```diff
@@ -27,7 +28,16 @@ async function registerScripts() {
     return;
   }
 
-  const domains: domainType[] = await api.settings.getAsync('customDomains');
+  let domains: domainType[] = await api.settings.getAsync('customDomains');
+
+  try {
+    const chibiRepo = await ChibiListRepository.getInstance().init();
+    const chibiDomains = chibiRepo.getPermissions();
+    domains = domains.concat(chibiDomains);
```

Comment:
Can you do that without `domains = ` or it does not work ?

Reply:
I just want the custom domains continue to work in case the chibi implementation throws an error.

Reply:
So are you saying that doing `domains.concat(chibiDomains);` without `domains =` wouldn't work ?

I'm not understanding why it wouldn't work, the error would be thrown before concatenating so it should be fine, am I wrong ?
Also the previous "customDomains" would still be present because they are added way before when creating the constant ? 

Am I missing something ?

Reply:
> > So are you saying that doing `domains.concat(chibiDomains);` without `domains =` wouldn't work ?
> > I'm not understanding why it wouldn't work, the error would be thrown before concatenating so it should be fine, am I wrong ? Also the previous "customDomains" would still be present because they are added way before when creating the constant ?
> > Am I missing something ?
> > _Sorry for focusing on this little detail 😅_
> 
> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat `concat()` does not modify existing arrays, it returns combined one, so to get both normal domains and chibi ones you have to assign result of `concat()` somewhere

Ohhh damn my bad x) Thank you lol

Reply:
I'll delete this then :)